src/cargo/ops/cargo_new.rs

```diff
@@ -158,12 +185,20 @@ fn global_config() -> CargoResult<CargoNewConfig> {
             })).0.to_string())
         }
     };
-    cfg.git = match cargo_new.get("git") {
+    cfg.version_control = match cargo_new.get("vcs") {
         None => None,
-        Some(git) => {
-            Some(try!(git.boolean().chain_error(|| {
-                internal("invalid configuration for key `cargo-new.git`")
-            })).0)
+        Some(vcs) => {
+            let vcs_str = try!(vcs.string().chain_error(|| {
+                internal("invalid configuration for key `cargo-new.vcs`")
+            })).0;
+            let version_control = match vcs_str.as_slice() {
+              "git" => VersionControl::Git,
+              "hg"  => VersionControl::Hg,
+              "none"=> VersionControl::NoVcs,
+              _  => return Err(internal("invalid configuration for key `cargo-new.vcs`")),
+            };
```

Comment:
Typically we use 4-space tabs for indentation, could you swap this and some parts up above to 4-space tabs?